minos/api_gateway/discovery/views/microservice.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from json import (
+    JSONDecodeError,
+)
+
+from aiohttp import (
+    web,
+)
+
+from ..database import (
+    MinosRedisClient,
+)
+from ..domain import (
+    Microservice,
+)
+from .router import (
+    routes,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@routes.view("/microservices/{name}")
+class MicroserviceView(web.View):
+    async def post(self):
+        body = await self.get_body()
+
+        redis_client = MinosRedisClient(config=self.request.app["config"])
```

Comment:
Why not to set the `MinosRedisClient` as a `web.Applicantion` configuration entry, instead of recreating it each time a request comes?

Reply:
This I'd rather deal with it in #43